src/main/java/org/myrobotlab/service/InMoov2Head.java

```diff
@@ -67,8 +67,10 @@ public InMoov2Head(String n, String id) {
     eyeX.setPin(22);
     eyeY.setPin(24);
     jaw.setPin(26);
-    //FIXME rollNeck must be connected to right controller on pin 12 not on pin 30
-    rollNeck.setPin(30);

```

Comment:
Where is rollNeck pin set if it's not being set here?

Reply:
InMoov2Head.java was not written correctly because it was estimated that all servos would be on left controller, but it is not correct. Some head servos like rollNeck and eyeLids are on the right controller, because they share the same number pins. Two Arduino with two NervoBoards.